layouts/partials/page/list.html

```diff
@@ -3,9 +3,10 @@
 {{ $pageLink := .Page.RelPermalink }}
 {{ $articleId := .Params.id | default .File.Path }}
 {{ $slug := (partial "common/slug" .) }}
-{{ $disableArticleBar := (eq (len .Content) 0) }}
+{{ $noContent := (eq (len .Content) 0) }}
+{{ $type := cond $noContent "index" "article" }}
 
-<div class="page index" data-roles="{{ $roles }}" id="{{ $uid }}" data-link="{{ $pageLink }}" permalink="{{ .Page.RelPermalink }}" data-disable-article-bar="{{ $disableArticleBar }}">
+<div class="{{ $type }} index" data-roles="{{ $roles }}" id="{{ $uid }}" data-link="{{ $pageLink }}" permalink="{{ .Page.RelPermalink }}" data-disable-article-bar="{{ $noContent }}" data-disable-article-options="{{ $noContent }}" data-disable-article-delete="true" >
```

Comment:
Nice.. I like the usage of article type.. wondering if that should rather be a data attribute too? Or do we use it for styling only?

Reply:
Oof, I would almost suggest we need to log a tech debt ticket for this. It's gone beyond the scope of this ticket.

Yes, in this case, `index` could be a data type instead. There's no styling in that class. The same goes for `page`. `article` however does have styling.

So I think these need cleanup but might need a bit of deeper think. We could change everything to data attributes, but anything that's an article would still need to be a class.

What do you think?

Reply:
yeah let's log a ticket and get back to this later. #fix-it-friday